backend/src/server.ts

```diff
@@ -1,6 +1,6 @@
 import { ENV } from '@common/enums/app/app';
 import { errorsHandler } from '@middlewares/middlewares';
-import { healthRouter, authRouter } from '@routes/routes';
+import { healthRouter, authRouter, activateRouter } from '@routes/routes';
```

Comment:
activate what? 
subscription? I don`t see what i activate from route. You need to rename it